### PR TITLE
[FR] Update `Game modifier`, `Half Time`, `Easy` and `Summary`

### PR DESCRIPTION
**Pull request by DoryanR**
Update of the translations of the following wiki pages:

- `Game_modifier/Summary`
- `Game_modifier`
- `Game_modifier/Half_Time`
- `Game_modifier/Easy`

Also:

- In `Game_modifier` (`en.md`, `es.md`, `id.md`, `ko.md`, `tr.md`) : add `(Experimental)` after `Cutting Edge` version

- [x] Need to update `Half Time` before review

### CODE REVIEW

File: wiki/Game_modifier/Summary/fr.md [hunk 1]
```diff
@@ -1,75 +1,73 @@
...
+| ![No Fail](/wiki/shared/mods/NF.png) | No Fail (NF) | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: 0.50x  | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: `Impossible  d'échouer, quoi que vous fassiez` |
+| ![Half Time](/wiki/shared/mods/HT.png) | Half Time (HT) | • ![][osu!] ![][osu!taiko] ![][osu!catch]: 0.30x • ![][osu!mania]: 0.50x | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: `Tout va plus lentement...` |
+| ![Hard Rock](/wiki/shared/mods/HR.png) | Hard Rock (HR) | • ![][osu!]: 1.12x ![][osu!taiko]: 1.06x • ![][osu!catch]: 1.12x • ![][osu!mania]: 1.00x | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: `Tout devient un peut plus compliqué.` |
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/Summary/fr.md:12]:
```suggestion
| ![Hard Rock](/wiki/shared/mods/HR.png) | Hard Rock (HR) | • ![][osu!]: 1.12x ![][osu!taiko]: 1.06x • ![][osu!catch]: 1.12x • ![][osu!mania]: 1.00x | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: `Tout devient un peu plus compliqué.` |
```

File: wiki/Game_modifier/Summary/fr.md [hunk 2]
```diff
@@ -1,75 +1,73 @@
...
+| ![Fade In](/wiki/shared/mods/FI.png) | Fade In (FI) | • ![][osu!mania]: 1.00x |  |
+| ![Flashlight](/wiki/shared/mods/FL.png) | Flashlight (FL) | • ![][osu!] ![][osu!taiko] ![][osu!catch]: 1.12x • ![][osu!mania]: 1.00x | • ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]: `Jouez dans la pénombre avec votre curseur en guise de lampe torche` |
+| ![Relax](/wiki/shared/mods/RL.png) | Relax (RL) | • ![][osu!] ![][osu!taiko] ![][osu!catch]: 0.00x | • ![][osu!]: `YVous n'avez pas besoin de cliquer. Vos doigts méritent une pause après toute cette agitation. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` • ![][osu!taiko]: `Détendez-vous ! Vous ne serez plus embêté par les spinners surprise, les drumrolls difficile ou les katus inattendus. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` • ![][osu!catch]: `Utilisez la souris pour contrôler le personnage. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` |
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/Summary/fr.md:20]:
```suggestion
| ![Relax](/wiki/shared/mods/RL.png) | Relax (RL) | • ![][osu!] ![][osu!taiko] ![][osu!catch]: 0.00x | • ![][osu!]: `YVous n'avez pas besoin de cliquer. Vos doigts méritent une pause après toute cette agitation. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` • ![][osu!taiko]: `Détendez-vous ! Vous ne serez plus embêté par les spinners surprise, les drumrolls difficiles ou les katus inattendus. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` • ![][osu!catch]: `Utilisez la souris pour contrôler le personnage. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` |
```

File: wiki/Game_modifier/Summary/fr.md [hunk 3]
```diff
@@ -1,75 +1,73 @@
...
+| ![Autopilot](/wiki/shared/mods/AP.png) | Auto Pilot (AP) | • ![][osu!]: 0.00x | • ![][osu!]: `Le mouvement du curseur est automatique, contentez-vous de suivre le rythme. ** LE SCORE RÉALISÉ NE SERA PAS COMPTABILISÉ **` |
+| ![Spun Out](/wiki/shared/mods/SO.png) | Spun Out (SO) | • ![][osu!]: 0.90x | • ![][osu!]: `Les spinners seront effectués automatiquement.` |
+| ![4K (*x*K)](/wiki/shared/mods/4K.png) | 4K (*x*K) | 0.66x–1.00x (voir [tableau ci-dessous](#multiplicateurs-de-score-pour-le-mod-xk)) |  |
+| ![Co-op](/wiki/shared/mods/CP.png) | Co-op (CP) | 0.70x–1.00x (voir [tableau ci-dessous](#multiplicateurs-de-score-avec-le-mod-*x*k-en-co-op)) | • ![][osu!mania]: ``Deux fois plus de touches pour deux fois plus de fun !` |
+| ![Mirror](/wiki/shared/mods/MR.png) | Mirror (MR) | • ![][osu!mania]: 1.00x |  |
+| ![Random](/wiki/shared/mods/RD.png) | Random (RD) | • ![][osu!mania]: 0.00x | • ![][osu!mania]: `Mettez un peut d'aléatoire pour pimenter votre partie.` |
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/Summary/fr.md:26]:
```suggestion
| ![Random](/wiki/shared/mods/RD.png) | Random (RD) | • ![][osu!mania]: 0.00x | • ![][osu!mania]: `Mettez un peu d'aléatoire pour pimenter votre partie.` |
```

File: wiki/Game_modifier/Summary/fr.md [hunk 4]
```diff
@@ -1,75 +1,73 @@
...
+| 4K | 0.78x | 0.82x | 0.86x | 1.00x | 0.90x | 0.90x | 0.90x | 0.90x | 0.90x |
+| 5K | 0.74x | 0.78x | 0.82x | 0.86x | 1.00x | 0.90x | 0.90x | 0.90x | 0.90x |
+| 6K | 0.70x | 0.74x | 0.78x | 0.82x | 0.86x | 1.00x | 0.90x | 0.90x | 0.90x |
+| 7K | 0.66x | 0.70x | 0.74x | 0.78x | 0.82x | 0.86x | 1.00x | 0.90x | 0.90x |
 
-*Note: Ce tableau s'applique uniquement aux beatmaps converties à partir d'osu! vers osu!mania par le jeu, et pas aux beatmaps créés spécifiquement pour osu!mania.*
+### Multiplicateurs de score avec le mod *x*K en Co-op
 
-Le Co-Op mod n'applique **aucune** pénalité sur les beatmaps spécifiquement créés pour osu!mania.
+*Remarque : Ce tableau n'affecte que les beatmaps convertis de osu! à osu!mania pendant le jeu, et non les beatmaps spécifiques à osu!mania. En tant que tel, le mod Co-op ne donne aucune pénalité sur les beatmaps spécifiques à osu!mania.*
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/Summary/fr.md:45]:
```suggestion
*Remarque : Ce tableau n'affecte que les beatmaps converties de osu! à osu!mania pendant le jeu, et non les beatmaps spécifiques à osu!mania. En tant que tel, le mod Co-op ne donne aucune pénalité sur les beatmaps spécifiques à osu!mania.*
```

File: wiki/Game_modifier/fr.md [hunk 5]
```diff
@@ -1,614 +1,72 @@
...
+## Liste des mods
 
-Ces mods peuvent faciliter le jeu au prix d'une diminution du multiplicateur de score.
+*Voir aussi : [Résumé (modificateur de jeu)](/wiki/Game_modifier/Summary)*
 
-### Easy
+Chacun des mods ci-dessous aura l'icône de son mode de jeu compatible (![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]) à côté d'elle.
 
-![Icône du mod Easy (EZ)](/wiki/shared/mods/EZ.png "Icône du mod Easy (EZ)")
+### Difficulté réduit
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/fr.md:30]:
```suggestion
### Difficulté réduite
```

File: wiki/Game_modifier/fr.md [hunk 6]
```diff
@@ -1,614 +1,72 @@
...
+Sur l'écran de sélection, les mods sont séparés en trois catégories : `Difficulté réduite`, `Difficulté augmentée`, et `Divers`. Ils peuvent soit réduire, augmenter ou supprimer le [`Multiplicateur de score`](/wiki/Score_multiplier). Lorsque deux mods sont utilisés simultanément, les multiplicateurs respectifs des mods seront multipliés ensemble (exemple, `1.06x * 1.12x = 1.1872x`).
 
-## Réduction des difficultés
+## Liste des mods
 
-Ces mods peuvent faciliter le jeu au prix d'une diminution du multiplicateur de score.
+*Voir aussi : [Résumé (modificateur de jeu)](/wiki/Game_modifier/Summary)*
 
-### Easy
+Chacun des mods ci-dessous aura l'icône de son mode de jeu compatible (![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]) à côté d'elle.
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/fr.md:28]:
```suggestion
Chacun des mods ci-dessous aura l'icône de son mode de jeu compatible (![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]) affiché sur le côté.
```

File: wiki/Game_modifier/fr.md [hunk 7]
```diff
@@ -1,614 +1,72 @@
...
+### Difficulté réduit
 
-- Multiplicateur de score : 0.50x
-- Touche de raccourci : `Q`
+- [Easy (EZ)](/wiki/Game_modifier/Easy) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [No Fail (NF)](/wiki/Game_modifier/No_Fail) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Half Time (HT)](/wiki/Game_modifier/Half_Time) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
 
-Le mod Easy divise par deux tous les paramètres de difficulté du beatmap sélectionné.
+### DDifficulté augmentée
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/fr.md:36]:
```suggestion
### Difficulté augmentée
```

File: wiki/Game_modifier/fr.md [hunk 8]
```diff
@@ -1,614 +1,72 @@
...
-- Touche de raccourci : -
-- Un mod pratique pour désactiver temporairement la vidéo lorsque cela est nécessaire.
-
-Le mod No Video désactive la vidéo de fond du beatmap.
-
-La fonctionnalité du mod a été déplacée vers [Visual Settings](/wiki/Visual_Settings) et l'icône n'est plus utilisée.
+### Autre
 
-## Anecdotes
+*Note : Ces mods ont été utilisés et disponibles il y a quelque temps, mais sont maintenant indisponibles.*
```
Comment (Ganondorf-1) [inline wiki/Game_modifier/fr.md:63]:
```suggestion
*Note : Ces mods étaient utilisés et disponibles il y a quelque temps, mais sont maintenant indisponibles.*
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by geshii — APPROVED** [review]

**Comment by DoryanR** [comment]
Can be merge

**Review by TicClick — APPROVED** [review]

**Pull request merged**